LocalSettings.php

```diff
@@ -3956,6 +3956,21 @@
 			'css' => [],
 		],
 	],
+	
+	// TextExtracts
+	'wgExtractsRemoveClasses' => [
+		'default' => false,
+		'gratispaideiawiki' => [
+			'.metadata', // This is a good thing to remove in any case
+			'span.coordinates',
+			'span.geo-multi-punct',
+			'span.geo-nondefault',
+			'#coordinates',
+		],
+	],
+	'wgExtractsExtendOpenSearchXml' => [
```

Comment:
once again, why are you setting the default only?

Reply:
Done

Reply:
No it isn't.

Reply:
How?